src/jit/gentree.h

```diff
@@ -932,7 +932,13 @@ struct GenTree
 #define GTF_CALL_POP_ARGS           0x04000000 // GT_CALL -- caller pop arguments?
 #define GTF_CALL_HOISTABLE          0x02000000 // GT_CALL -- call is hoistable
 #ifdef LEGACY_BACKEND
+#ifdef _TARGET_ARM_
+// Since trashable registers have to be preserved on PollGC helper we don't set this flag for ARM
```

Comment:
I might add a bit more detail on this, something like:
```
// The GTF_CALL_REG_SAVE flag indicates that the call preserves all integer registers. This is used for
// the PollGC helper. However, since the PollGC helper on ARM follows the standard calling convention,
// for that target we don't use this flag.
```